starknet-rpc-test/src/lib.rs

```diff
@@ -20,10 +30,22 @@ use url::Url;
 pub struct MadaraClient {
     process: Child,
     client: Client,
-    rpc_request_count: Cell<usize>,
+    rpc_request_count: AtomicUsize,
```

Comment:
You plan to share the struct between threads?

Reply:
to have it lazy loaded it's necessary yes even if we run tests with one thread in practice

Reply:
Still required with fixtures?